sidebars.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
         "label": "Getting Started",
         "items": [
           "getting-started",
+          "Okteto License",
```

Comment:
I think this should be `okteto-license`

Reply:
```suggestion
          "okteto-license",
```

You're right, I got confused entering some version of "Okteto License" in 5 different places lol